about.md

```diff
@@ -6,10 +6,4 @@ permalink: /about/
 
 Welcome to the Literature Review site of the [Videos & Images Theory and Analytics Laboratory (VITAL) of Sherbrooke University](http://vital.dinf.usherbrooke.ca/), headed by Professor Pierre-Marc Jodoin. This lab focuses on researching state of the art algorithms in image processing, computer vision, video analytics, medical imaging and statistical models. If you’d be interested in collaborating with us, do not hesitate to [contact us](http://vital.dinf.usherbrooke.ca/contact/)!
```

Comment:
Welcome to the Literature Review site of the [Videos & Images Theory and Analytics Laboratory (VITAL) of Sherbrooke University](http://vital.dinf.usherbrooke.ca/), headed by Professor Pierre-Marc Jodoin(http://info.usherbrooke.ca/pmjodoin).  Our lab specializes on  image processing, computer vision, video analytics, medical imaging and statistical modeling. If you’d be interested in collaborating with us, do not hesitate to [contact us](http://vital.dinf.usherbrooke.ca/contact/)!